lizard_auth_server/forms.py

```diff
@@ -42,6 +46,40 @@ def clean(self):
         return new_data
 
 
+class JWTDecryptForm(forms.Form):
+    key = forms.CharField(max_length=1024)
+    message = forms.CharField(max_length=8192)
+
+    def clean(self):
+        """Verifies the JWT from the site and cleans form data.
+
+        Note: replaces the original form data with JWT payload, which should
```

Comment:
Dit vervangen... is dat wel zo netjes? Zeker omdat de inhoud van de message vervolgens niet gecontroleerd wordt.